CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 ## 4.77.0 (Unreleased)
 
+NOTES:
+* vpcaccess: reverted the ability to update the number of instances for resource `google_vpc_access_connector` ([#15313](https://github.com/hashicorp/terraform-provider-google/pull/15313))
+
+FEATURES:
+* **New Resource:** `google_document_ai_warehouse_document_schema` ([#15326](https://github.com/hashicorp/terraform-provider-google/pull/15326))
+* **New Resource:** `google_document_ai_warehouse_location` ([#15326](https://github.com/hashicorp/terraform-provider-google/pull/15326))
+
+IMPROVEMENTS:
+* alloydb: added `continuous_backup_config` and `continuous_backup_info` fields to `cluster` resource ([#15370](https://github.com/hashicorp/terraform-provider-google/pull/15370))
+* cloudbuild: removed the validation function for the values of `machine_type` field on the `google_cloudbuild_trigger` resource ([#15357](https://github.com/hashicorp/terraform-provider-google/pull/15357))
+* compute: add future_limit in quota exceeded error details for compute resources. ([#15346](https://github.com/hashicorp/terraform-provider-google/pull/15346))
+* compute: added `ipv6EndpointType` and `ipVersion` to `google_compute_address` ([#15358](https://github.com/hashicorp/terraform-provider-google/pull/15358))
+* compute: added `local_ssd_recovery_timeout` field to `google_compute_instance` resource ([#15366](https://github.com/hashicorp/terraform-provider-google/pull/15366))
+* compute: added `local_ssd_recovery_timeout` field to `google_compute_instance_template` resource ([#15366](https://github.com/hashicorp/terraform-provider-google/pull/15366))
+* compute: added `network_interface.ipv6_access_config.external_ipv6_prefix_length` to `google_compute_instance` ([#15358](https://github.com/hashicorp/terraform-provider-google/pull/15358))
+* compute: added `network_interface.ipv6_access_config.name` to `google_compute_instance` ([#15358](https://github.com/hashicorp/terraform-provider-google/pull/15358))
+* compute: added a new type `GLOBAL_MANAGED_PROXY` for the filed `purpose` in the resource `google_compute_subnetwork` ([#15345](https://github.com/hashicorp/terraform-provider-google/pull/15345))
+* compute: added field `instance_lifecycle_policy` to `google_compute_instance_group_manager` and `google_compute_region_instance_group_manager` (ga) ([#15322](https://github.com/hashicorp/terraform-provider-google/pull/15322))
+* compute: added protocol type: UNSPECIFIED in `google_compute_backend_service` as per [release note](https://cloud.google.com/load-balancing/docs/release-notes#July_24_2023)
+ ([#15328](https://github.com/hashicorp/terraform-provider-google/pull/15328))
+* compute: made `network_interface.ipv6_access_config.external_ipv6` configurable in `google_compute_instance` ([#15358](https://github.com/hashicorp/terraform-provider-google/pull/15358))
```

Comment:
Should this be in the more common form of: added x field to y resource?

Reply:
The field existed before as output-only, so I don't think that would work here. Open to better wording though

Reply:
Oh weird, ok